sonar-plugin-api/src/main/java/org/sonar/api/utils/System2.java

```diff
@@ -69,6 +69,16 @@
   public static final System2 INSTANCE = new System2();
 
   /**
+   * Same as {@link #INSTANCE} but naming is more appropriate for static imports.
+   * @since 5.5
+   */
+  public static final System2 SYSTEM = INSTANCE;
+
+  protected System2() {
```

Comment:
no issue in SQ for missing comment here?

why not use `private` (and put a comment saying "use INSTANCE" or "prevents direct instanciation") ?